presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/execution/SqlFunctionExecutors.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.functionNamespace.execution;
+
+import com.facebook.presto.functionNamespace.execution.thrift.ThriftSqlFunctionExecutor;
+import com.facebook.presto.spi.function.FunctionImplementationType;
+import com.facebook.presto.spi.function.RoutineCharacteristics.Language;
+import com.google.inject.Inject;
+
+import java.util.Map;
+import java.util.Set;
+
+import static java.util.Objects.requireNonNull;
+
+public class SqlFunctionExecutors
```

Comment:
curious: why this is called `SqlFunctionExecutors` instead of `SqlFunctionExecutor`? Looks like there is just one `ThriftSqlFunctionExecutor` inside ? 😂

Reply:
Since execution is configurable, and Presto communication is primarily in http there could potentially be a http executor as well. Or maybe if we want to add other RPC protocol later. Calling it "executors" instead of "executor" just so we don't need to rename it in case any of those happens 😂